editor.planx.uk/src/@planx/components/PropertyInformation/Public.tsx

```diff
@@ -45,7 +45,17 @@ function Component(props: PublicProps<PropertyInformation>) {
       titleBoundary={passport.data?.["property.boundary.title"]}
       blpuCodes={blpuCodes}
       overrideAnswer={overrideAnswer}
-      handleSubmit={props.handleSubmit}
+      handleSubmit={() => {
+        // If the user changed their property type, they'll already have a previous PropertyInformation breadcrumb that set `_overrides`
+        const hasOverrodeAnswer = passport.data?.["_overrides"]?.["property.type"];
+        const passportData = {
+          "propertyInformation.action": hasOverrodeAnswer ? "Changed the property type" : "Accepted the property type",
+        };
```

Comment:
There's a bit of a tricky edge case possible here - "Accepted the property type" isn't _really_ semantically correct if `showPropertyTypeOverride` wasn't enabled to begin with - but I think a binary makes most sense for analytics clarity here rather than introducing a third possible track like "Proceeded through Property Information" ? 

Any objections?

Reply:
Thanks for flagging this - agree with your decision here. 

I think a binary choice here is clearer and easier to understand and manage than something that's maybe semantically perfect 👍